src/types/payload.ts

```diff
@@ -229,6 +232,7 @@ export const PayloadSchema = Type.Object({
   repository: RepositorySchema,
   organization: Type.Optional(OrganizationSchema),
   installation: Type.Optional(InstallationSchema),
+  repositories_added: Type.Optional(Type.Array(RepositorySchema)),
```

Comment:
It seems not right. the property should be a part of [`WebhookPayloadWithRepository`](https://github.com/probot/probot/blob/952f88b/src/context.ts#L177). Would you be making sure the property exists in `WebhookPayloadWithRepository` if I am missing something.

Reply:
> WebhookPayloadWithRepository

It doesn't exist

Reply:
I want to keep the comment unresolved until things get clear. would you be clarifying why you added it `PayloadSchema` even if you think its in `RepositorySchema`.

Reply:
> I want to keep the comment unresolved until things get clear. would you be clarifying why you added it `PayloadSchema` even if you think its in `RepositorySchema`.

I don't understand this

Reply:
`repositories_added` contains objects that matches the `RepositorySchema`, so i am using it

Reply:
besides `repository` had a type `RepositorySchema` anyway and this new field is just like an array of repositories